sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj

```diff
@@ -35,4 +35,9 @@
     <Compile Include="$(AzureCoreSharedSources)TaskExtensions.cs" LinkBase="Shared" />
   </ItemGroup>
 
+  <!-- Temporary fix to get Test Proxy to work around Core issue that has not yet been shipped -->
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
```

Comment:
Is this required for all the projects? I don't like leaving all these ProjectReferences in all these projects for a long period like over the holidays. If this is needed I'd like us to publish a patch of Azure.Core soon to get these removed so that we don't keep them very long.

Reply:
Don't we have a release freeze until Jan?

Reply:
Not for patches - though the changes to Core probably warrant a new minor version.

Reply:
The bigger issue is that if we change all these to ProjectReference we will block all these projects from doing a release, at least from main. I'm not a fan of having ProjectReferernces long so we should see about when we can get a new Azure.Core released with the changes before we take this PR.

Reply:
Alright, I won't merge this until we figure out when we can do a Core release.

Reply:
Let's ignore those headers until we get core shipped

Reply:
This is not currently supported in the Test Proxy /cc @scbedd

Reply:
Well it will be!